Makefile

```diff
@@ -291,18 +294,19 @@ bundle-register:
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& az acr login --name $${ACR_NAME}	\
+	&& . ./devops/scripts/get_access_token.sh \
 	&& cd ${DIR} \
 	&& ${ROOTPATH}/devops/scripts/register_bundle_with_api.sh --acr-name "$${ACR_NAME}" --bundle-type "$${BUNDLE_TYPE}" --current --insecure --tre_url "$${TRE_URL:-https://$${TRE_ID}.$${LOCATION}.cloudapp.azure.com}" --verify --workspace-service-name "$${WORKSPACE_SERVICE_NAME}"
 
-shared-service-register-and-deploy:
+deploy-shared-service:
```

Comment:
Would it be better if we...

- leave the `shared-service-register-and-deploy` rule in place
- have it call the new rule
- have it output a message saying "this rule is deprecated...please use deploy-shared-service instead"

... and then raise an issue to remove this rule later? It allows us to have a gap between people pulling this change, and then fixing their AzDO builds